src/js/components/DataTable/Searcher.js

```diff
@@ -77,7 +77,7 @@ const Searcher = ({
       {filters[property] ? (
         <Box
           flex={false}
-          pad={{ horizontal: 'small' }}
+          pad={theme.dataTable.searcher?.text?.pad}
```

Comment:
```suggestion
          pad={theme.dataTable.search?.text?.pad}
```
feels like "searcher" is the internal component but we don't really use that kind of naming elsewhere. Should we just call "search"?

Reply:
Yes, search makes more sense from a user's perspective, I'll update